op-e2e/system_test.go

```diff
@@ -617,7 +617,6 @@ func L1InfoFromState(ctx context.Context, contract *bindings.L1Block, l2Number *
 // TestSystemMockP2P sets up a L1 Geth node, a rollup node, and a L2 geth node and then confirms that
 // the nodes can sync L2 blocks before they are confirmed on L1.
 func TestSystemMockP2P(t *testing.T) {
-	t.Skip("flaky in CI") // TODO(CLI-3859): Re-enable this test.
```

Comment:
Is this going to be flaky still?

Reply:
It passed every time I ran it, as well as in CI, so I think it's good. If flaky again we can disable it. But while modifying the unsafe-block processing, closely related to gossip, I think it's good to enable now.